packages/manager/modules/vps/src/upscale/translations/Messages_fr_FR.json

```diff
@@ -1,31 +1,40 @@
 {
+  "vps_upscale_range_capacity": "de {{ min }} à {{ max }} Go",
+  "vps_upscale_from_on": "À partir de <span class=\"vps-upscale-model-price__price\">{{price}}</span>/mois",
```

Comment:
i don't like using classes in translations, is it possible that you use only standard html tags and use css selectors to style your price ?

Reply:
Yes I'll remove them

Reply:
Bad idea, if you remove the class and target the `span`, the day you add a `span` in the translation, you will have an unwanted side effect

Reply:
@FredericEspiau  class + span won't be removed, it will be extracted into the html, and pass through `translate-values` directive.